src/systems/core/message-actions/create-update.js

```diff
@@ -452,6 +452,18 @@ export default class CreateUpdate {
             // handle some special cases for attributes (e.g. attributes set directly to the THREE.js object);
             // default is to let aframe handle attributes directly
             switch (attribute) {
+                // Defer until physics is loaded. If it never loads...trigger load??
```

Comment:
I like the loading of physics based on the presence of static/dynamic body.

Reply:
Oops, maybe I should have taken this out entirely for a specific physics set of changes